src/main/java/fr/insee/genesis/configuration/Config.java

```diff
@@ -19,4 +19,7 @@ public class Config {
 	@Value("${fr.insee.genesis.sourcefolder.specifications}")
 	private String specFolderSource;
 
+	@Value("${fr.insee.genesis.logfolder}")
```

Comment:
Maybe we can have the same parsing fr.insee.properties.log.filename and not create a new config